aiida/requirements.txt

```diff
@@ -1 +1 @@
-aiida-core~=1.6.1,!=1.6.2
+aiida-core~=2.5
```

Comment:
add a newline please - alternatively, you could also put this in the `environment.yml` to keep all the dependencies in the same place.

Reply:
Good point! I'll move it there.

Reply:
Not sure why, but `conda` installs AiiDA v2.2, despite the current default version on `conda-forge` being 2.5.1... Anyway, I pinned the version and everything seems to be fine, at least with the Docker image I created locally and pushed to the registry :)

Reply:
oh I meant you could still use `pip` but inside the `environment.yml` - either way is fine

Reply:
Alright, I think I figured out why it does that. v2.5.1 is not compatible with the base image due to the `erlang` version. I therefore pinned `aiida-core.services` to v2.2.2 (which would also be the default selected by `conda`), and am installing `aiida-core` at v2.5.1 via `pip` instead right now. Would be nice not to have this mixing, but it seems to work, and `renku` is also updated via `pip` down the line, so I guess it's the best setup for now :)
Pinning both versions should make it stable, at least for the foreseeable future.